elementary/monitor/cli.py

```diff
@@ -115,6 +115,7 @@ def monitor(
         slack_webhook,
         slack_token,
         slack_channel_name,
+		  slack_timezone,
```

Comment:
Please make sure to align the parameter with the others.

Reply:
Don't know why, but I see it aligned throught the VS-Code.

Reply:
No problem, I added suggestions accordingly.